src/modules/user/user.service.ts

```diff
@@ -0,0 +1,15 @@
+import { Injectable } from '@nestjs/common';
+import { User } from '../../../entities/User.entity';
+import { UserDAO } from './user.dao';
+@Injectable()
+export class UserService {
+  constructor(private readonly userDao: UserDAO) {}
+
+  async getUserInfoById(userId: string): Promise<User> {
+    return this.userDao.getUserInfoById(userId);
+  }
+
+  async getUserByEmail(email: string): Promise<User> {
+    return this.userDao.getUserByEmail(email);
+  }
```

Comment:
@matheo97 is there a service that creates the user? shouldn't we include it among this functions as well?

Reply:
We don't have it but we definetely need it, I propose you to approve this PR and I can work on the CRUD of users as my next task, this way you guys will have the 'session' already available when working in your endpoints.